internal/v1mcp/tools/crem.go

```diff
@@ -39,15 +39,17 @@ func toolCREMAttackSurfaceDevicesList(client *v1client.V1ApiClient) mcpserver.Se
 			mcp.WithString("filter", mcp.Description(tooldescriptions.FilterAttackSurfaceDevices)),
 			mcp.WithString("orderBy",
 				mcp.Enum(
-					"latestRiskScore asc",
-					"latestRiskScore desc",
-					"deviceName asc",
-					"deviceName desc",
+					withOrdering(
+						asc_desc,
+						"deviceName",
+						"latestRiskScore",
+					)...,
 				),
 				mcp.Description("The field by which the results are sorted"),
 			),
-			mcp.WithNumber("top",
+			mcp.WithString("top",
 				mcp.Description(tooldescriptions.DefaultTop),
+				mcp.Enum(cremTop()...),
```

Comment:
❓ Here `mcp.Description` comes first in this method, which is the same as the other files like `cam`, though it is the opposite order to the rest of this file. 

Does the order of the arguments matter?